lib/realm.dart

```diff
@@ -1,3 +1,10 @@
+/// Realm SDK for Flutter and Dart
```

Comment:
copyright header should be at the top.

Reply:
It is the same like the previous comment. The slashes ////////////// appear in the doc. I moved this line to prevent this.